ci/ray_ci/automation/ray_wheels_lib.py

```diff
@@ -0,0 +1,133 @@
+import boto3
+from typing import List, Tuple
+import os
+from botocore.exceptions import ClientError
+
+from ci.ray_ci.utils import logger
+
+bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
+
+PYTHON_VERSIONS = ["cp38-cp38", "cp39-cp39", "cp310-cp310", "cp311-cp311"]
+PLATFORMS = [
+    "manylinux2014_x86_64",
+    "manylinux2014_aarch64",
+    "macosx_10_15_x86_64",
+    "macosx_11_0_arm64",
+    "win_amd64",
+]
+RAY_TYPES = ["ray", "ray_cpp"]
+
+
+def _check_downloaded_wheels(
+    directory_path: str, wheels: List[str]
+) -> Tuple[bool, str]:
+    """
+    Check if the wheels are downloaded as expected.
+
+    Args:
+        directory_path: The directory where the wheels are downloaded and stored in.
+        wheels: The list of wheel names to check.
+
+    Returns:
+        A tuple of (bool, str):
+            If the wheels are downloaded properly, the first element
+            is True and the second element is the message.
+            If the wheels are not downloaded properly, the first element is
+            False and the second element is the error message.
+    """
+
+    if not os.path.exists(directory_path):
+        return False, f"{directory_path} does not exist"
```

Comment:
discussed offline, for these errors that not worth continuing the operation, just use assertion and fail here. let's keep it simple.